api/v2/serializers/post/token_update.py

```diff
@@ -23,30 +24,61 @@ def validate(self, data):
         IF identity is found validation will:
         - Ensure that user/token produces a valid driver
         """
-        validated_data = data
-        self.validate_token_with_driver(data['provider'], data['username'], data['project_name'], data['token'])
-        return validated_data
+        logger.info("TokenUpdateSerializer:validate called")
+        # create will fail if it is an invalid token, but we need to create an identity first
```

Comment:
this function helps us check whether the token is valid and whether we can use it to create a libcloud driver so later on, we could use this token. Why do we have to get rid of it?